src/index.html

```diff
@@ -5,9 +5,58 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
```

Comment:
don't forget about code style, fix it in all cases
you can check code style - [here](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)
![image](https://github.com/mate-academy/layout_product-cards/assets/91411692/fca06495-f6ae-4511-ac88-c4f220d631f7)
```suggestion
    <link
      rel="preconnect"
      href="https://fonts.gstatic.com"
      crossorigin="anonymous"
    >
```